--- ./gio/gdebugcontrollerdbus.c.orig
+++ ./gio/gdebugcontrollerdbus.c
@@ -357,9 +357,9 @@ authorize_cb (GObject      *object,
 
   if (!authorized)
     {
-      GError *local_error = g_error_new (G_DBUS_ERROR, G_DBUS_ERROR_ACCESS_DENIED,
-                                         _("Not authorized to change debug settings"));
-      g_dbus_method_invocation_take_error (invocation, g_steal_pointer (&local_error));
+//      GError *local_error = g_error_new (G_DBUS_ERROR, G_DBUS_ERROR_ACCESS_DENIED,
+//                                         _("Not authorized to change debug settings"));
+//      g_dbus_method_invocation_take_error (invocation, g_steal_pointer (&local_error));
     }
   else
     {
